src/components/taxonomy/sub-topic.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components'
+
+import { TopicCategories } from './topic-categories'
+import { Link } from '@/components/content/link'
+import { TaxonomyData, TaxonomySubTerm } from '@/data-types'
+import { renderArticle } from '@/schema/article-renderer'
+
+export interface TopicProps {
+  data: TaxonomyData
+}
+
+export interface SubTopicProps {
+  data: TaxonomySubTerm
+  subid: number
+  id: number
+}
+
+export function SubTopic({ data, subid, id }: SubTopicProps) {
+  return (
+    <>
+      <h2 className="my-3 serlo-h2 border-0" style={{ hyphens: 'auto' }}>
+        <Link href={data.url} path={[subid, 'title']}>
+          {data.title}
+        </Link>
+      </h2>
+
+      <Wrapper className="mb-5 pb-6 flex flex-col sm:flex-row border-b-1 border-gray-300">
```

Comment:
small note: `border-b` is the same as `border-b-1` (not adding a number is defaulting to 1px, so we can do this without changing the theme.)